### DISCUSSION TITLE
默认播放器样式-宽屏变换缓慢

### DISCUSSION THREAD

**Discussion by 971230** [the1812/Bilibili-Evolved#4681] [提问 / Q&A] [answered]
就是点开一个视频，会先是按照默认的【常规】来播放视频，要过一会儿才会变成【宽屏】显示视频内容，这个是bug还是专门设置的，还是那里可以调整这个时间间隔
![PixPin_2024-03-14_21-11-31](https://github.com/the1812/Bilibili-Evolved/assets/37500200/400af6b4-c9c0-471f-b6c9-0ade8c8428dd)
因为GIF上传大小限制10MB，图截屏了好几次，只能这样局部了

**Comment by 971230** [comment]
![PixPin_2024-03-14_21-11-31](https://github.com/the1812/Bilibili-Evolved/assets/37500200/0d5d63f9-fb4a-4902-be4e-37bc17f28223)

**Comment by the1812** [comment, marked as answer]
默认播放器模式早就不维护了

**Reply by 971230** [reply]
啊这，大清亡了我现在才知道🤣
